concepts/src/app.ts

```diff
@@ -1,6 +1,31 @@
 import express from "express";
+import type { Client as ElasticClient } from "@elastic/elasticsearch";
 
-const app = express();
+type Clients = {
+  elastic: ElasticClient;
+};
+
+const createApp = (clients: Clients) => {
+  const app = express();
+
+  app.get("/concepts", (req, res) => {
+    res.status(200).json({
+      type: "ResultList",
+      results: Object.values(concepts),
+    });
+  });
+
+  app.get("/concepts/:id", (req, res) => {
+    const concept = concepts[req.params.id];
+    if (concept) {
+      res.status(200).json(concept);
+    } else {
+      res.sendStatus(404);
```

Comment:
We certainly don't need it for 1.0, but seeing this makes me wonder if we'll ever need to support 302 Redirect/410 Gone in this API.